### PR TITLE
fix faust compile

### PR DESCRIPTION
**Pull request by cchafe**
Fixes 
‘cerr’ is not a member of ‘std’ 
found with dev-test-faust-compile
-- does not fix warnings about unused parameters
-- noted that Faust compiler used for the currently checked-in C++ is very old (from 2017)

### CODE REVIEW

**Comment by cchafe** [comment]
fails to build -- anyone know why?

File: src/compressordsp.h [hunk 1]
```diff
@@ -311,36 +381,40 @@ class dsp_factory
 #ifndef __meta__
 #define __meta__
 
+/**
+ The base class of Meta handler to be used in dsp::metadata(Meta* m) method to retrieve
+ (key, value) metadata.
+ */
 struct Meta {
-    virtual ~Meta(){};
+    virtual ~Meta() {}
```
Comment (ntonnaett) [inline src/compressordsp.h:389]: This looks sketchy.

Comment (ntonnaett) [reply src/compressordsp.h:389]: No. That's right. It's late…

File: src/makeXcodeproj.sh [hunk 2]
```diff
@@ -1 +1 @@
-qmake -spec macx-xcode ../jacktrip.pro
+qmake - spec macx - xcode../ jacktrip.pro
```
Comment (ntonnaett) [inline src/makeXcodeproj.sh:1]: This doesn't look right. Is it clang-format on a shell script?

Comment (josmithiii) [reply src/makeXcodeproj.sh:1]: It works on my Mac to remove the extra spaces:

```qmake -spec macx-xcode ../jacktrip.pro```

Comment (josmithiii) [reply src/makeXcodeproj.sh:1]: But note that it generates ```jacktrip.xcodeproj``` in the src directory.

**Comment by cchafe** [comment]
starting over

**Pull request closed without merging**